contracts/test/ClaimTest.t.sol

```diff
@@ -578,4 +621,281 @@ contract ClaimTest is AddressRegistry, StateManipulations, TestUtils {
         // Fails to claim after loan is closed
         try pool.claim(address(loanV2), address(debtLockerFactory)) { assertTrue(false, "Able to claim"); } catch { }
     }
+
+    function test_claim_defaultUncollateralized() external {
```

Comment:
For these more basic tests, we should actually assert that the entire BPT burning flow works

Reply:
Just for the uncollateralized and undercollateralized basic tests. Once we assert that its working with the claim amounts we're fine for the rest of the tests

Reply:
You can refer to the maple-core V1 integration tests for how to assert the BPT burning flow and LP losses